integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -0,0 +1,83 @@
+const _ = require('lodash/fp')
+const pRetry = require('p-retry')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, delay, dismissNotifications, findElement, findText, fillIn, input, select, signIntoTerra, waitForNoSpinners, findInGrid, navChild } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const snapshotName = 'testsnapshot'
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
+
+const testRunWorkflowOnSnapshotFn = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ dataRepoUrl, environment, page, testUrl, snapshotColumnName, snapshotId, snapshotTableName, token, workflowName, workspaceName }) => {
+  if ( environment == "perf") {
```

Comment:
Rather than wrapping this in a whole big if statement, what if we just return early? Like this:

```js
  if (environment === 'perf') {
    return
  }
```

Also, please make sure to use `===` instead of `==`. While we don't currently have a way to enforce lint on integration tests, you can run `npx eslint integration-tests/tests --fix` from the root of the repo to clean things up.